lib/index.js

```diff
@@ -1,19 +1,48 @@
 // This file is in the entry point in your webpack config.
 import './styles.scss';
 import CurrentWeather from './current-weather'
+import BackgroundImage from './background-image'
 
-$('form').on('submit', function(event) {
+$(document).ready(function() {
+  $('#location-form').on('submit', getLocationInfo);
+  $('#sign-up').on('submit', registerUser);
+})
+
+const registerUser = (event) => {
+  event.preventDefault();
+  $('.errors').text('');
+  const email = $('input')[0].value;
+  const password = $('input')[1].value;
+  const confPassword = $('input')[2].value;
+  if(!password || !confPassword || password !== confPassword) {
```

Comment:
Your password verification here makes sense to me and is totally valid as is. A couple other thoughts if you want to make it even more user-friendly...

One thing you could do to simplify this statement is on your `<input>` elements in your HTML, you can add a field of `required` which doesn't allow the user to submit the form if those parameters are empty. Then you don't need to worry about checking if they are blank in your Javascript.

For checking if the password and confPassword fields match, you could also add on change event listeners to your password and confPassword elements to do "live" check if they match as the user types and don't allow the user to submit the form until they match.

Reply:
Ah, the `required` makes it so much cleaner! Thank you!